R/reactives.R

```diff
@@ -1583,6 +1708,20 @@ maskReactiveContext <- function(expr) {
 #' @param ignoreNULL Whether the action should be triggered (or value
 #'   calculated, in the case of \code{eventReactive}) when the input is
 #'   \code{NULL}. See Details.
+#' @param skipFirst Whether the action should be run the first time the
```

Comment:
I like `ignoreInitial`, especially because of keeping the same vocab as `ignoreNULL`. But I feel like `ignoreFirst` rolls off the tongue better. Is there a particular reason you're leaning towards "initial" instead of "first", @jcheng5?